### PR TITLE
Issues 1700

### PR DESCRIPTION
**Pull request by raftaar1191**
## Description
PR for issues #1700 

## How Has This Been Tested?
Manually tested

### CODE REVIEW

**Comment by raftaar1191** [comment]
@DevinWalker  @ravinderk 

Please review this PR in merge

File: includes/admin/tools/data/class-give-tools-delete-donors.php [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+	}
+
+	/**
+	 * Return the calculated completion percentage.
+	 *
+	 * @since 1.8.12
+	 * @return int
+	 */
+	public function get_percentage_complete() {
+		if ( 1 == $this->step ) {
```
Comment (ravinderk) [inline includes/admin/tools/data/class-give-tools-delete-donors.php]: @raftaar1191 why it is hardcoded. It should be dynamic, right?

Comment (DevinWalker) [inline includes/admin/tools/data/class-give-tools-delete-donors.php]: Yeah this should chunk it out depending on how many donors fit the steps.

File: includes/admin/tools/data/class-give-tools-delete-donors.php [hunk 2]
```diff
@@ -0,0 +1,353 @@
+<?php
```
Comment (DevinWalker) [inline includes/admin/tools/data/class-give-tools-delete-donors.php]: Can you rename this file to `class-give-tools-delete-test-donors.php`

Comment (raftaar1191) [inline includes/admin/tools/data/class-give-tools-delete-donors.php]: Done

**Comment by raftaar1191** [comment]
@DevinWalker  @ravinderk 

Please kindly review the PR and merge

**Comment by DevinWalker** [comment]
@raftaar1191 can you fix the merge conflicts here?

**Comment by raftaar1191** [comment]
@DevinWalker 

Solved the conflicts in https://github.com/WordImpress/Give/pull/1897/commits/202a52ea941e7412349aaa6f38c1ac3cb03c7938

**Pull request merged**